From d235c929ca0162b70179412f804f5fc4bcdf46b6 Mon Sep 17 00:00:00 2001
From: HougeLangley <hougelangley1987@gmail.com>
Date: Sat, 31 Jul 2021 17:41:12 +0800
Subject: [PATCH] replace and test

---
 src/platforms/linux/blurhelper/windowblur.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/platforms/linux/blurhelper/windowblur.h b/src/platforms/linux/blurhelper/windowblur.h
index ae86cdb..f2e8a6d 100644
--- a/src/platforms/linux/blurhelper/windowblur.h
+++ b/src/platforms/linux/blurhelper/windowblur.h
@@ -20,7 +20,7 @@
 #ifndef WINDOWBLUR_H
 #define WINDOWBLUR_H
 
-#include <QApplication>
+#include <QtWidgets/QApplication>
 #include <QObject>
 #include <QQmlEngine>
 #include <QQmlParserStatus>
-- 
2.32.0

